tests/test_settings.py

```diff
@@ -15,3 +18,40 @@ def test_import_error_message_maintained(self):
         })
         with self.assertRaises(ImportError):
             settings.DEFAULT_RENDERER_CLASSES
+
+    def test_bad_iterable_setting_class_raises_warning(self):
+        """
+        Make sure warnings are emitted when settings which should be iterable
+        are not.
+        """
+        settings = APISettings({
+            'DEFAULT_RENDERER_CLASSES': 'rest_framework.renderers.JSONRenderer'
+        })
+
+        with warnings.catch_warnings(record=True) as w:
```

Comment:
I'm not really sure what you mean by this comment. Should I be adding another test of some sort, or changing this one?